app/matplotlib.ipynb

```diff
@@ -47,21 +48,36 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
    "metadata": {
     "collapsed": true,
     "ExecuteTime": {
-     "end_time": "2023-07-07T12:36:31.669207Z",
-     "start_time": "2023-07-07T12:36:31.661449Z"
+     "end_time": "2024-11-07T15:50:17.255407Z",
+     "start_time": "2024-11-07T15:50:17.170091Z"
     }
    },
-   "outputs": [],
    "source": [
     "years = [2015, 2016, 2017, 2018, 2019]\n",
     "population = [100, 120, 150, 180, 200]\n",
     "\n",
-    "# write your code here"
-   ]
+    "plt.plot(years, population)\n",
+    "plt.xlabel(\"Year\")\n",
+    "plt.ylabel(\"Popluation\")\n",
```

Comment:
There is a typo in the y-axis label 'Popluation'. It should be corrected to 'Population'.